.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -82,7 +82,7 @@ chown -R root:"${DAEMON_GROUP}" ${SUBMITTY_INSTALL_DIR}/autograder
 chmod -R 750 ${SUBMITTY_INSTALL_DIR}/autograder
 
 if [ "${WORKER}" == 1 ]; then
-    chown -R root:${SUPERVISOR_USER} ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils
+    chown -R ${SUPERVISOR_USER} ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils
```

Comment:
Please use `user:group` notation here as we do everywhere else, even if the intention is that it'll be `${SUPERVISOR_USER}:${SUPERVISOR_USER}`. This keeps things consistent, and also very explicit on what the permissions are.